### ISSUE TITLE
UnusedUses incorrectly reports some types used in unions

### ISSUE THREAD

**Issue by Majkl578** [slevomat/coding-standard#280]
```php
<?php

declare(strict_types=1);

namespace Test;

use Doctrine\Common\Collections\Collection;
use Doctrine\ORM\Mapping\Property;
use ProxyManager\Proxy\GhostObjectInterface;

class Foo
{
    /**
     * @param iterable|Property[] $propertyMappings
     * @param array|Collection|object[] $collection The collection.
     * @return object|GhostObjectInterface|null The entity reference.
     */
    public function bar($propertyMappings, $collection) {}
}
```
The sniff reports:
```
 7 | ERROR | [x] Type Doctrine\Common\Collections\Collection is not used in this file. (SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse)
 8 | ERROR | [x] Type Doctrine\ORM\Mapping\Property is not used in this file. (SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse)
 9 | ERROR | [x] Type ProxyManager\Proxy\GhostObjectInterface is not used in this file. (SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse)
```

Only dev-master, 4.4.4 is ok.

**Comment by kukulich** [comment]
I'm sorry, should be fixed in https://github.com/slevomat/coding-standard/commit/bfda8d10556d31a18b3c16d2a420ac30f2a09c07

**Comment by Majkl578** [comment]
Looks ok, thanks. 👍
